test/api_stability/CMakeLists.txt

```diff
@@ -0,0 +1,46 @@
+cmake_minimum_required(VERSION 3.8)
+
+project (api_stability)
+
+find_package(Git)
+
+include (../seqan3-test.cmake)
+
+include (ExternalProject)
+set (SEQAN3_LAST_STABLE_VERSION "3.0.2")
```

Comment:
This is not the same as the package version because we configure this test with the current master, yes?

I wonder if this could be automated (ask GitHub for the latest tag)?
But even then you would need to adapt the checksum...

Since this it is probably easy to miss when bumping the version, it should be mentioned in the release template (edit the current release task issue).

Reply:
I added a TODO in release task